schemas/Manifest.yaml

```diff
@@ -7,8 +7,9 @@ properties:
   manifestItems:
     type: array
     example:
-      - "qr.example.com"
-      - "qr.test.com"
+      - "https://qr.example.com/i/123465678"
```

Comment:
Yes, agree we should have a good example here. Want to also suggest we make sure to be clear about documentation re: how QR's interact with .well-known via parameter serialization standards: https://swagger.io/docs/specification/serialization/